frontend/src/components/ChatBox.js

```diff
@@ -0,0 +1,116 @@
+import { useState, useEffect, useRef } from 'react'
+import { Container , Card, InputGroup, Button, Form } from 'react-bootstrap'
+
+const ChatBox = ({ socket, username, room }) => {
+    const [messagesReceived, setMessagesReceived] = useState([])
+    const [message, setMessage] = useState('')
+    const chatBody = useRef(null)
+    const chatInput = useRef(null)
```

Comment:
What is the useRef hook used for?

Reply:
Allows me to access DOM elements by individually selecting/referencing them

![image](https://user-images.githubusercontent.com/77485021/195420617-498fccf0-8437-45a7-8772-f23a7c3b2b4e.png)
![image](https://user-images.githubusercontent.com/77485021/195420706-b9e16254-f2ad-46bd-a655-397f77bf50f0.png)
![image](https://user-images.githubusercontent.com/77485021/195420865-f8df4325-684a-40e8-a92e-6e814f808dd6.png)

Reply:
Ok